### PR TITLE
Feature/us120 qtr end to reset scoreboard qtr start time

### PR DESCRIPTION
**Pull request by hayrich**
@wraus/scoreboard-devs

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -7,7 +7,7 @@
     <groupId>au.org.sports.wrugby</groupId>
     <artifactId>scoreboard</artifactId>
     <version>1.0-SNAPSHOT</version>
-    <packaging>war</packaging>
+    <packaging>jar</packaging>
```
Comment (david2202) [inline pom.xml]: Need to revert this change.  war packaging is correct for spring-boot and then run with java -jar scoreboard-1.0-SNAPSHOT.war.  The war file contains the embedded tomcat.

**Comment by hayrich** [comment]
reverted change to pom packaging, now using war
